datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -325,6 +339,57 @@ private static void configureMlModelResolvers(final RuntimeWiring.Builder builde
             );
     }
 
+    /**
+     * Configures resolvers responsible for resolving the {@link com.linkedin.datahub.graphql.generated.DataFlow} type.
+     */
+    private static void configureDataFlowResolvers(final RuntimeWiring.Builder builder) {
+        builder
+            .type("Owner", typeWiring -> typeWiring
+                .dataFetcher("owner", new AuthenticatedResolver<>(
+                    new LoadableTypeResolver<>(
+                        CORP_USER_TYPE,
+                        (env) -> ((Owner) env.getSource()).getOwner().getUrn()))
+                )
+            );
+    }
```

Comment:
You shouldn't need this wiring- ``owner`: Owner` type is already configured in dataset's configure-resolvers method and dataflow + datajobs are able to re-use that

Reply:
Thanks, I'll remove it! I got confused as I was following the `configureMlModelResolvers` example, where it was configured as well. Should I remove it?

Reply:
@frsann I would double check using graphiql ui before making that change but I believe that should be fine

cc @jjoyce0510 to confirm

Reply:
Got it, I'll check with the IDE

Reply:
Gabe's correct - we should be able to remove this. We should likely refactor the `configureDatasetResolvers` method to extract configuration of the "Owner" type into a dedicated method. For now, feel free to remove it here and with the ML models, I just ask that you issue a test query to verify that we can still traverse to the Owner type!

Reply:
Thanks, will do!